src/car/simple/SimpleCar.hpp

```diff
@@ -37,13 +39,13 @@ class SimpleCar : public Car
      * car.setSpeed(50); // Car with half speed forward
      * \endcode
      */
-    void setSpeed(float speed) override;
+    virtual void setSpeed(float speed) override;
```

Comment:
:warning: **modernize-use-override** :warning:
'virtual' is redundant since the function is already declared 'override'
```suggestion
    virtual void ```